shared/haml/course_explorer_table.haml

```diff
@@ -6,7 +6,7 @@
   :css
     .courses .bar-#{n} {
       float: left;
-      width: #{n * 72}px;
+      width: #{n * 74.5}px;
```

Comment:
Interesting.  Does the subpixel rendering become obvious anywhere?

Reply:
My understanding is that subpixel rending is a problem in like IE 5/6, but not modern browsers we support.

Reply:
Cool, just wanted to make sure there weren't any visual artefacts because of the subpixel values.